hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminEndpoint.java

```diff
@@ -147,6 +145,9 @@ RSGroupInfoManager getGroupInfoManager() {
     return groupInfoManager;
   }
 
+  RSGroupAdminServer getGroupAdminServer() {
```

Comment:
Please provide @VisibleForTesting annotation.

Reply:
thank you for the tip